features/step_definitions/customer_booking_request_steps.rb

```diff
@@ -91,6 +91,37 @@
   @step_two.accessibility_requirements.set false
 end
 
+When(/^I enter an email address with a typo$/) do
+  with_mock_mailgun_response(is_valid: false,
```

Comment:
I'll get rubocop validating this but for now can we indent the args in this style please:

```ruby
method_name(
  key: value,
  key: value
)
```